Test Debug Tutorial/README.md

```diff
@@ -0,0 +1,5 @@
+### Testing the Debug Tutorial
+Test the [Debugging Tutorial](/docs/system/tutorials/debugtutorial/index.html) by creating Tests from Rule Autopsies. Run the Tests and Test Suite to ensure that the Rules 
+work as desired.
+
+For more information on the Test Debugging Tutorial, check out the documentation [here](/docs/system/tutorials/testdebugtutorial/index.html).
```

Comment:
I don't think the relative URLs in these READMEs work from the Github repos  (they would only work from the generated docs hosted on the vantiq server).
Instead you probably need to hard-code a public server like dev.vantiq.com
```suggestion
For more information on the Test Debugging Tutorial, check out the documentation [here](https://dev.vantiq.com/docs/system/tutorials/testdebugtutorial/index.html).
```

Reply:
The relative links work because we render the README files in the context of a user's browser that is already pointing at an instance of the IDE. To test it out, go to any instance and go to Projects > Import > Tutorials and choose the Client Builder tutorial and click the "here" link at the end. The links don't work when clicked in Github, but I think that's ok given they are only intended to be used in the import popup.